packages/xo-server/src/xapi/index.js

```diff
@@ -598,6 +610,13 @@ export default class Xapi extends XapiBase {
         tags,
         version: asInteger(version),
         xenstore_data,
+
+        // VM created Suspended
+        power_state: suspend_VDI !== undefined ? 'Suspended' : undefined,
+        suspend_VDI,
+        domain_type,
+        last_boot_CPU_flags,
+        last_booted_record,
```

Comment:
These fields do not appear to be supported by `VM.create`, at least they are not documented: https://xapi-project.github.io/xen-api/classes/vm.html

Reply:
Yes I added them in my PR in the XAPI to support the suspended creation.

Reply:
How did it behave previously?

Reply:
The VM was created Halted and they were given default values